roles/keystone-defaults/defaults/main.yml

```diff
@@ -83,6 +83,7 @@ keystone:
     sensu_checks:
       check_keystone_api:
         criticality: 'critical'
+    cron_bin_path:  "/sbin:/usr/sbin:{{ (os == 'rhel' ) | ternary('/usr/bin/', '/usr/local/bin/') }}"
```

Comment:
So do you think I should just put inside the template:
```
PATH=/sbin:/usr/sbin:/usr/bin/:/usr/local/bin/
```
because it'll search in all of those paths on ubuntu and rhel?
keystone manage is in /usr/bin in rhel and `ip` is in /sbin or /usr/sbin